starter-core/src/main/java/io/micronaut/starter/feature/build/maven/templates/pom.rocker.raw

```diff
@@ -593,7 +600,7 @@ List<Property> properties
             <include>**/*Test.*</include>
           </includes>
         </configuration>
-@if (features.testFramework().isKotlinTest()) {
+@if (features.testFramework().isKotlinTest() || features.testFramework().isKoTest()) {
```

Comment:
This could be simpler if this PR was merged: https://github.com/micronaut-projects/micronaut-starter/pull/411